Formula/newsboat.rb

```diff
@@ -12,6 +12,7 @@ class Newsboat < Formula
   end
 
   depends_on "asciidoc" => :build
```

Comment:
This can be removed now, we only use Asciidoctor now.

Reply:
Cool!